pkg/eval/vals/string.go

```diff
@@ -20,6 +20,8 @@ func ToString(v interface{}) string {
 	case int:
 		return strconv.Itoa(v)
 	case float64:
+		// Float64 is special-cased for historical reasons. The other number types are handled by
+		// the Stringer interface.
```

Comment:
Not really for historical reasons; it's because I don't like any of Go's builtin ways to print float64. The point about other number types are correct.